source/mapping-rules/introduction.txt

```diff
@@ -105,6 +108,34 @@ Mapping rules can be defined with the following ``Migrate as`` options:
    * - :ref:`Migrate rows as Embedded documents <rm-embedded-documents>`
      - Translates each row as a set of fields or an embedded object in a child document identified by a foreign key relationship.
 
+Custom Fields and Filters
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can create mapping rules with :ref:`custom fields <rm-field-customizations>`. 
+Relational Migrator also supports :ref:`table filters <rm-table-filters>` to limit 
+data migrated from relational tables.
+
+Synthetic Foreign Keys
+----------------------
+
+You can create synthetic foreign keys in Relational Migrator to define a logical 
+relationship between your relational database tables. For more information,
+see :ref:`<rm-synthetic-foreign-keys>`.
+
+Preview Mapping Rules
+---------------------
+
+.. include:: /includes/mapping-options-preview.rst
+
+Learn More
+----------
+
+- For information on mapping your relational schema to a MongoDB schema, 
+  see :ref:`<rm-schema-mapping>`.
+- For information about mapping rule options, see :ref:`<rm-mapping-rule-options>`.
+- For information about the diagram that displays your relational and 
+  MongoDB schemas, see :ref:`rm-diagrams`.
```

Comment:
Optional - but I think just the links with the page titles would suffice here, rather than having "For more information..."

```suggestion
- :ref:`rm-schema-mapping`.
- :ref:`rm-mapping-rule-options`.
- :ref:`rm-diagrams`.
```